ios/BufferModule.swift

```diff
@@ -0,0 +1,89 @@
+import BitmovinPlayer
+
+@objc(BufferModule)
+public class BufferModule: NSObject, RCTBridgeModule {
+    /**
+     * Collection of `BufferLevel` objects
+     * - Parameter audio: `BufferLevel` for `MediaType.Audio`.
+     * - Parameter video: `BufferLevel` for `MediaType.Video`.
+     */
+    internal struct RNBufferLevels {
+        let audio: BufferLevel
+        let video: BufferLevel
+    }
```

Comment:
Just figured we already have RN-only types on the bottom of `RCTConvert+BitmovinPlayer.swift`, so please move this there.

Reply:
Address in fb87736f34f23a5ec852cf4a9c8d1117a9683bfd